.github/workflows/main.yml

```diff
@@ -0,0 +1,17 @@
+name: Lighthouse CI
+on: push
+jobs:
+    lighthouseci:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+            - uses: actions/setup-node@v3
+              with:
+                  node-version: '18'
+                  cache: 'yarn'
+                  registry-url: 'https://registry.npmjs.org'
+            - run: npm install -g @lhci/cli@0.12.x
```

Comment:
Since lighthouse is installed as part of the monorepo, you shouldn't need to install it separately here. The yarn install that follows will include the lighthouse cli.